### ISSUE TITLE
invidio.us feature parity

### ISSUE THREAD

**Issue by mauriceoegerli** [mauriceoegerli/viewtube-vue#2]
| Feature | Invidious | ViewTube |
| - | - | - |
| Audio-only mode | X |  |
| No need to keep window open on mobile) | X | X |
| Free software (AGPLv3 licensed) | X | X |
| No ads | X | X |
| No need to create a Google account to save subscriptions | X | X |
| Lightweight (homepage is ~4 KB compressed) | X | N |
| Tools for managing subscriptions: | X |  |
| Only show unseen videos | X |  |
| Only show latest (or latest unseen) video from each channel | X |  |
| Delivers notifications from all subscribed channels | X |  |
| Automatically redirect homepage to feed | X | X |
| Import subscriptions from YouTube | X |  |
| Dark/Light mode | X |  |
| Embed support | X |  |
| Set default player options (speed, quality, autoplay, loop) | X |  |
| Does not require JS to play videos | X |  |
| Support for Reddit comments in place of YT comments | X |  |
| Import/Export subscriptions, watch history, preferences | X |  |
| Does not use any of the official YouTube APIs | X | X |
| Developer API | X | N |

N: Not intended

**Comment by davidcollini** [comment]
Per-Channel RSS support should also be added to this list

**Comment by davidcollini** [comment]
Resume watched video playback where it ended should also be added, Reference [#372 In Piped](https://github.com/TeamPiped/Piped/issues/372)

**Comment by davidcollini** [comment]
Also settings don’t work without JavaScript

**Comment by davidcollini** [comment]
I believe this project would be the best if it had feature parody with every YouTube frontend, so here’s features from Piped and CloudTube that are great

Piped
Lbry Integration

CloudTube 
Subscribe to channels without an account

**Comment by moisout** [comment]
> Resume watched video playback where it ended should also be added, Reference [#372 In Piped](https://github.com/TeamPiped/Piped/issues/372)

This works when using an account

**Comment by moisout** [comment]
> I believe this project would be the best if it had feature parody with every YouTube frontend, so here’s features from Piped and CloudTube that are great
> 
> Piped
> Lbry Integration
> 
> CloudTube 
> Subscribe to channels without an account

I'll add it, but priority right now is getting it to be more stable and reliable

**Comment by davidcollini** [comment]
> I'll add it, but priority right now is getting it to be more stable and reliable

Of course! I just really love this project, and wanted to know if you'd be open to implementing these features since I want to completely move away from using invidious, piped and CloudTube for different tasks, but working on speed and stability is much better than adding features that won't be reliable, thank you and this is the most ambitious frontend for YouTube I've seen!

**Comment by moisout** [comment]
> > I'll add it, but priority right now is getting it to be more stable and reliable
> 
> Of course! I just really love this project, and wanted to know if you'd be open to implementing these features since I want to completely move away from using invidious, piped and CloudTube for different tasks, but working on speed and stability is much better than adding features that won't be reliable, thank you and this is the most ambitious frontend for YouTube I've seen!

Thanks for the kind words!

**Comment by sdfg2** [comment]
First: this is amazing.  I started with invidious, moved to piped, then found this this morning and it answers most of the problems I had with the other two, so thank you so much for your effort!  I really want to help out where I can with this, I've already had a few issues with setting it all up, but I've been doing my due diligence and searching existing issues first and came across this.

Invidious had the 'only show unseen' for subscriptions (which you have listed), and had a 'watched' indicator when going to the actual channel (which isn't listed).  One thing I would love is to have a 'watched' indicator present everywhere, and a toggleable option for 'only show unseen' for the feed.

**Comment by Jogai** [comment]
Would it make sense to extend the comparison with piped?

**Comment by arjpar** [comment]
I agree with @Jogai to also have a table featuring Piped parity. Off the top of my head, I can't think of what Piped has that Invidious doesn't though.
